aptos-move/framework/aptos-framework/sources/storage_gas.move

```diff
@@ -0,0 +1,359 @@
+module aptos_framework::storage_gas {
+
+    use aptos_framework::system_addresses;
+    use std::error;
+    use aptos_framework::state_storage;
+    use std::vector;
+
+    friend aptos_framework::gas_schedule;
+    friend aptos_framework::genesis;
+    friend aptos_framework::reconfiguration;
+
+    const ESTORAGE_GAS_CONFIG: u64 = 0;
```

Comment:
These are not visible to end users, but can we still add comments here explaining what these error codes are for?

Reply:
to me it is quite self-explanatory. But I can add some.